ioctl/newcmd/hdwallet/hdwalletimport.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package hdwallet
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tyler-smith/go-bip39"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/pkg/util/fileutil"
+)
+
+// Multi-language support
+var (
+	_importCmdShorts = map[config.Language]string{
+		config.English: "import hdwallet using mnemonic",
+		config.Chinese: "通过助记词导入钱包",
+	}
+	_importCmdUses = map[config.Language]string{
+		config.English: "import",
+		config.Chinese: "import 导入",
+	}
+)
+
+// NewHdwalletImportCmd represents the hdwallet import command
+func NewHdwalletImportCmd(client ioctl.Client) *cobra.Command {
+	use, _ := client.SelectTranslation(_importCmdUses)
+	short, _ := client.SelectTranslation(_importCmdShorts)
+
+	cmd := &cobra.Command{
+		Use:   use,
+		Short: short,
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			if fileutil.FileExists(_hdWalletConfigFile) {
+				cmd.Println("Please run 'ioctl hdwallet delete' before import")
+				return nil
+			}
+
+			cmd.Println("Enter 12 mnemonic words you saved, separated by space")
+
+			line, err := client.ReadSecret()
```

Comment:
client.ReadSecret  cannot see the input character.  this is not right.  see https://github.com/iotexproject/iotex-core/blob/773fb82140af8c7d4bf81e38b323e6eec3e75da7/ioctl/cmd/hdwallet/hdwalletimport.go#L57-L61
move these codes into client interface. and call function here